src/app/tests/suites/TestGroupKeyManagementCluster.yaml

```diff
@@ -625,6 +690,24 @@ tests:
                         EpochStartTime2: 2110002,
                     }
 
+    - label: "KeySet Write 2 TrustFirst"
+      command: "KeySetWrite"
+      PICS: "!GRPKEY.S.F00"
+      arguments:
+          values:
+              - name: "GroupKeySet"
+                value:
+                    {
+                        GroupKeySetID: 0x01a2,
+                        GroupKeySecurityPolicy: 0,
+                        EpochKey0: "\xd0\xd1\xd2\xd3\xd4\xd5\xd6\xd7\xd8\xd9\xda\xdb\xdc\xdd\xde\xdf",
+                        EpochStartTime0: 2110000,
```

Comment:
Suggest coverage also with a value of 0 for epoch 0 time which is a legal and very common value.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/26752 to track this.  Not quite sure what more needs to be done other than having some command that sets EpochStartTime0 to 0.... @tcarmelveilleux Please drop any relevant thoughts in that issue?